pydoc-markdown/src/pydoc_markdown/contrib/renderers/docusaurus.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf8 -*-
+from nr.databind.core import Field
+from nr.interface import implements, override
+from pathlib import Path
+from pydoc_markdown.interfaces import Renderer
+from pydoc_markdown.contrib.renderers.markdown import MarkdownRenderer
+from typing import Any, Dict, List, Text
+import docspec
+import json
+import os.path
+
+
+@implements(Renderer)
+class DocusaurusRenderer(MarkdownRenderer):
```

Comment:
Could you instead make the `MarkdownRenderer` a field named `markdown` in the `DocusaurusRenderer`? That would be in line with how the `HugoRenderer` and `MarkdownRenderer` do it.